src/ProviderHub/test/Get-AzProviderHubCustomRollout.Tests.ps1

```diff
@@ -0,0 +1,26 @@
+$loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
```

Comment:
Just noticed that you do not implement any tests and recording the test. And they are required.

Reply:
Thanks for catching that. It seems the tests were overwritten after re-generating the cmdlets. I've re-incorporated them and ran the tests again.